docs/topics/DatabaseOperations.adoc

```diff
@@ -316,12 +366,17 @@ ykpersonalize -2 -a -ochal-resp -ochal-hmac -ohmac-lt64 -oserial-api-visible -oa
 ```
 
 You will be asked to enter the HMAC key you created earlier, copy/paste they key output in the first step. Repeat both steps for your second YubiKey. We recommend storing your HMAC key in a safe place (e.g., printed on paper) in case you need to recreate another key.
+
+== Command Line Tool
+KeePassXC comes with a robust command line tool *keepassxc-cli* to access, view, and manipulate your database directly from any terminal window. The tool is separately documented through it's man page which can be shown using `man keepassxc-cli` or through the on-demand help using `keepassxc-cli [command] -h`. An online version of the man page is accessible https://github.com/keepassxreboot/keepassxc/blob/master/docs/man/keepassxc-cli.1.adoc[here].
 // end::advanced[]
 
 == Storing a Database File
 The database file that you create might contain highly sensitive data and must be stored in a very secure way. You must make sure that the database is always protected with a strong and long password. The database file that is protected with a strong and long password is secure and encrypted while stored on your computer or cloud storage service.
 
-Make sure that the database file is stored in a folder that is secure. Make sure that you or someone else does not accidentally delete the database file. Deletion of the database file will result in the total loss of your information and a lot of inconvenience to manually retrieve your logins for  various web applications. You must not share your database file with anyone unless absolutely necessary.
+Make sure that you or someone else does not accidentally delete the database file. Deletion of the database file will result in the total loss of your information and a lot of inconvenience to manually retrieve your logins for various web applications. We do not recommend sharing your database file with anyone unless you absolutely trust them (spouse, child, etc).

```

Comment:
Sharing is not well defined here. I know you just modified the text a bit but I assume the idea here is sharing the same file between two or more people knowing the same shared secret to open it.